test/powershell/Language/Scripting/Debugging/DebuggerCommand.Tests.ps1

```diff
@@ -0,0 +1,424 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+Describe 'Basic debugger command tests' -tag 'CI' {
+
+    BeforeAll {
+        Register-DebuggerHandler
+    }
+
+    AfterAll {
+        Unregister-DebuggerHandler
+    }
+
+    Context 'Help (?, h) command should display the debugger help message' {
+        BeforeAll {
+            $testScript = {
+                try {
+                    $bp = Set-PSBreakpoint -Command Get-Process
+                    Get-Process -Id $PID > $null
+                } finally {
+                    Remove-PSBreakPoint -Breakpoint $bp
+                }
+            }
+
+            $results = @(Test-Debugger -ScriptBlock $testScript -CommandQueue '?','h')
+            $result? = if ($results.Count -gt 0) {$results[0].Output -join [Environment]::NewLine}
+            $resulth = if ($results.Count -gt 1) {$results[1].Output -join [Environment]::NewLine}
```

Comment:
`$result?`, `$resulth` - can we use more informative names? Also at first look I was confused with ? in name.

Reply:
Hmmm. The commands allow me to capture the result from specific commands, and store the result of those commands in a variable that follows the pattern result_commandName_. By using these names instead of result1, result2, result3, I can see at a glance what result I am checking in other parts of the test without having to look back to see which number represents what test. Maybe I'll use a hashtable for the results instead. (e.g. `$result['s']`, `$result['?']`, etc.). That's probably easier to follow.

Reply:
Done.